HOWTO-RELEASE

```diff
@@ -32,47 +45,191 @@
+
+If needed, update the year in `CITATION`.
+
+*Commit the changes to master.*
+
+
+1.5 Create maintenance branch
+-------------------------------------------------------------------------------
+
+*Skip this step if you are preparing a patch release or RC2 and later.*
+
+Create the branch:
+
+```
+git branch x.y
+```
+
+where `x` is the major version number and `y` is the minor version number. Bug
+fixes found in the release candidates should be pushed to master and
+subsequently cherry-picked to the maintenance branch.
+
+*Push branch upstream.*
+
+
+1.6 Prepare and upload archives
+-------------------------------------------------------------------------------
+
+Run autoconf and configure the build:
+
+```
+./autogen.sh
+mkdir build
+cd build
+../configure
+make dist
+```
+
+Rename the archives and generate MD5 sum files:
+
+```
+mv proj-x.y.z.tar.gz proj-x.y.zRCn.tar.gz
+mv proj-x.y.z.zip proj-x.y.zRCn.zip
+md5 proj-x.y.zRCn.tar.gz > proj-x.y.zRCn.tar.gz.md5
+md5 proj-x.y.zRCn.zip > proj-x.y.zRCn.zip.md5
+```
+
+Upload to the OSGeo download server:
+
+```
+scp proj-x.y.zRCn.* warmerdam@download.osgeo.org:/osgeo/download/proj
+```
+
+Adjust version numbers and usernames accordingly.
+
+
+1.7 Announce the release candidate
+-------------------------------------------------------------------------------
+
+Announce the release candidate on the PROJ mailing list.
+
+
+1.8 Promotion to final release
+-------------------------------------------------------------------------------
+
+When you are confident that the latest release candidate is ready for promotion
+to final release a motion for promotion should be issued on the PROJ mailing
+list. Allow for a minimum of 48 hours for voting time.
+
+
+2 Final release
+===============================================================================
+
+Barring a successfull promotion process we can proceed with the final release.
+
+2.1 Prepare and upload archives
+-------------------------------------------------------------------------------
+
+See step 1.6 above. Do not rename the archives with RC postfixes.
+
+2.2 Update website
+------------------------------------------------------------------------------
+
+    - Update the version number in `doc/source/conf.py`
+    - Add the release notes from `NEWS` to `docs/source/index.rst`
+    - Update download links in `docs/source/download.rst`
+
+*Commit the changes to master and cherrypick into maintenance branch.*
 
-5) Update man pages:
+When pushed upstream the website will be update to the current version.
 
-    ./scripts/update_man.sh
+2.3 Tag the release
+-------------------------------------------------------------------------------
 
-   The script builds man pages from Sphinx docs, copies them to man/ and commits
-   them to git.
+Tag the release with:
 
-6) Add a note to the ChangeLog that a new release is being issued, and what
-   the release number is.
+```
+git tag x.y.z
```

Comment:
An annotated tag is better. ```git tag -a -m "Create tag x.y.z" x.y.z```

Reply:
Did not know about that feature. Updated.